sdk/src/azure/core/az_http_header_validation_private.h

```diff
@@ -56,18 +56,18 @@ static const uint8_t az_http_valid_token[256] = {
   0, // 31 ->  unit separator
   ' ', // 32 ->  space
   '!', // 33 ->  !
-  0, // 34 ->  “
+  0, // 34 ->  "
   '#', // 35 ->  #
   '$', // 36 ->  $
   '%', // 37 ->  %
   '&', // 38 ->  &
-  '\'', // 39 ->  ‘
+  '\'', // 39 -> '
   0, // 40 ->  (
   0, // 41 ->  )
   '*', // 42 ->  *
   '+', // 43 ->  +
   0, // 44 ->  ,
-  '-', // 45 ->  –
+  '-', // 45 ->  -
```

Comment:
@vhvb1989 - did you verify that we don't have any other non-ASCII characters in the file?

Reply:
I did. 
in VSCode we can re-open file with an specific encoding.  I checked there is no other weird encoding

Reply:
Also, this returned no results. So all is good:

> grep --color='auto' -P -n "[\x80-\xFF]" sdk/src/azure/core/az_http_header_validation_private.h